cmd/gen-go-sample/main_test.go

```diff
@@ -299,6 +347,61 @@ func initTestGenerator() *generator {
 	}
 }
 
+func labelp(l descriptor.FieldDescriptorProto_Label) *descriptor.FieldDescriptorProto_Label {
+	return &l
+}
+
+// mapTypeAndField creates the generated MapEntry protobuf message and the actually map field.
+// If valueType is enum or message, vTypeName must not be empty.
+func mapTypeAndField(
```

Comment:
nit: this will read better if you  precede it with a verb, eg `createMapTypeAndField`